scripts/build/libc/glibc.sh

```diff
@@ -102,7 +103,14 @@ do_libc_backend() {
     # If gcc is not configured for multilib, it still prints
     # a single line for the default settings
     multilibs=( $("${CT_TARGET}-gcc" -print-multi-lib 2>/dev/null) )
+    last_multi=
     for multilib in "${multilibs[@]}"; do
+        last_multi=$(( ${#multilibs[@]} - 1 ))
+        if [ "${multilib%%;*}" = "${multilibs[last_multi]%%;*}" ]; then
+            # This is the last multilib build or multilib is '.'
+            # (default target, not multilib)
+            multi_last=y
+        fi
```

Comment:
This is a bit convoluted. If you'd build it with the first multilib, the code
would be so much simpler!

    first=1
    for multilib blabla; do
        blabla all code
        if [ ${first} -eq 1 ]; then
            blabla build manuals...
        fi
        first=0
    done

Et voilà! ;-)